controllers/apply-next/simple/fields.test.js

```diff
@@ -264,40 +264,6 @@ describe('fields', () => {
         });
     }
 
-    describe('companyNumber', () => {
```

Comment:
These tests have been combined with the related ones in `form.test.js` where we can test the normalised error messages, not just `joi` behaviour.